src/internet_identity/src/http.rs

```diff
@@ -253,6 +214,61 @@ fn security_headers() -> Vec<HeaderField> {
     ]
 }
 
+/// Full content security policy delivered via HTTP response header.
+///
+/// This policy also includes the `frame-ancestors` directive in addition to the policies included in the HTML `meta` tag.
+/// We deliver the CSP by header _and_ meta tag because the headers are not yet certified.
```

Comment:
~~Why don't we just deliver it in the meta tag then?~~

Ah, looks like the `meta` tag doesn't support all directives.

Reply:
Yes, exactly.